### ISSUE TITLE
Title: WNGrad: Learn the Learning Rate in Gradient Descent

### ISSUE THREAD

**Issue by ikedaosushi** [ikedaosushi/tech-news#1043]
Title: WNGrad: Learn the Learning Rate in Gradient Descent<br>
Authors: Xiaoxia Wu, Rachel Ward, L&eacute;on Bottou Abstract: Adjusting the learning rate schedule in stochastic gradient methods is an important unresolved problem which requires tuning in practice.<br>
http://ift.tt/2Dh0mK9
